Documentation/admission-controller-usage.md

```diff
@@ -12,21 +12,12 @@ Enabling the Rook admission controller is recommended to provide an additional l
 
 ## Quick Start
```

Comment:
How about if we remove this topic, and create a new section in the pre-requisites (pre-reqs.md) page? I'm thinking after the Ceph prereqs section, we would have a new section called `## Admission Controller`. We would explain that it's recommended, but optional to first install the cert manager to enable the admission controller.

Reply:
done